provisioner/roles/control_node/templates/ansible.cfg.j2

```diff
@@ -7,6 +7,8 @@ host_key_checking = False
 retry_files_enabled = False
 {% if (workshop_type == 'rhel') or (workshop_type == 'security') %}
 inventory = /home/{{username}}/lab_inventory/hosts
+{% elif workshop_type == 'devops' %}  
```

Comment:
curious on your opinion if we should start moving this all to `workshop` and simplify these templates down :) . or if the name is helpful here